### PR TITLE
refactor: Improve log in seeding 

### PR DESCRIPTION
**Pull request by CarloVarni**
Adding some `VERBOSE` statements in the  seeding, from Grid creation and Filling to seed finder and filter.

### CODE REVIEW

File: Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.ipp [hunk 1]
```diff
@@ -39,7 +43,7 @@ Acts::CylindricalSpacePointGridCreator::createGrid(
                               // = pT[MeV] / (300 *Bz[kT])
 
     // sanity check: if yOuter takes the square root of a negative number
-    if (minHelixRadius < config.rMax / 2) {
+    if (minHelixRadius < config.rMax * 0.5) {
```
Comment (andiwand) [inline Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.ipp:46]: unrelated change?

**Review by andiwand — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :white_check_mark: Athena integration test results [0c57839b5b469b4ab585fe6a8254f06bb086ce7b]
### :white_check_mark: All tests successful

|status|job|report|
|------|---|------|
|:green_circle: | [report_pull_request](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/45356513)||
|:green_circle: | [run_art_test: test_run4_acts_vertex_PU200](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/45356512)||
|:green_circle: | [run_art_test: test_run4_ttbar_PU200](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/45356511)||
|:green_circle: | [run_art_test: test_data18_13TeV_1000evt](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/45356510)||
|:green_circle: | [run_art_test: test_ttbarPU40_reco](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/45356509)||
